lib/process-services-cloud/src/lib/people/components/people-cloud.component.ts

```diff
@@ -316,15 +316,22 @@ export class PeopleCloudComponent implements OnInit, OnChanges, OnDestroy {
         this.isLoading = false;
     }
 
-    isPreselectedUserValid(preselectedUser: IdentityUserModel, validatedUser: IdentityUserModel) {
-        if (validatedUser && (validatedUser.id !== undefined || validatedUser.username !== undefined || validatedUser.email !== undefined)) {
-            return preselectedUser.id === validatedUser.id || preselectedUser.username === validatedUser.username || preselectedUser.email === validatedUser.email;
-        } else {
-            return false;
+    isPreselectUserAlignedWithValidatedUser(preselectedUser: IdentityUserModel, validatedUser: IdentityUserModel) {
```

Comment:
can you please apply a different strategy such as:
```compare(preselecte, identity) {
        Object.keys(preselecte).forEach( (property) => {
            if (typeof property !== 'undefined') {
                return identity[property]  === preselecte[property];
            }
        })
    }

Reply:
in case the order of the properties is not correct change the inteface
```
export interface IdentityUserModel {
    id?: string;
    username?: string;
    email?: string;
    firstName?: string;
    lastName?: string;
    createdTimestamp?: any;
    emailVerified?: boolean;
    enabled?: boolean;
    readonly?: boolean;
}
```